.github/workflows/pullrequest-trusted.yml

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
[note] RDK link https://github.com/viamrobotics/rdk/blob/main/.github/workflows/pullrequest-trusted.yml

Reply:
Just to be clear, all instances of `qaworkflow` also have to be changed to main before merging right?

Reply:
mind please posting the diff?